profiles/killbill/src/main/java/org/killbill/billing/server/filters/RequestDataFilter.java

```diff
@@ -32,12 +34,13 @@
 @Singleton
 public class RequestDataFilter implements ContainerRequestFilter, ContainerResponseFilter {
 
+    private static final String REQUEST_ID_HEADER = "X-Request-Id";
 
```

Comment:
For future reference, according to the HTTP spec, headers are case insensitive, so either way should work.